client/gutenberg/extensions/slideshow/index.js

```diff
@@ -11,6 +11,7 @@ import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
  */
 import edit from './edit';
 import save from './save';
+import { transforms } from './transforms';
```

Comment:
Nitpick, why not use `export default` in `transforms.js`, and `import transforms from `./transforms` here?

Reply:
Good call. Done in https://github.com/Automattic/wp-calypso/pull/30636/commits/d03a577b422b8d2b5085be8a0fb64c90c139a508